kerncraft/kernel.py

```diff
@@ -513,19 +511,25 @@ def print_constants_info(self, output_file=sys.stdout):
             table += '{!s:>8} | {:<10}\n'.format(name, value)
         print(prefix_indent('constants: ', table), file=output_file)
 
+    def iaca_analysis(self, *args, **kwargs):
+        raise NotImplementedError("Kernel does not support compilation and iaca analysis. "
+                                  "Try a different model e.g. ECMData")
```

Comment:
The model does not choose which kernel class will be used, that is bases on the arguments passed to kerncraft.py